content/usage/basic.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Basic command examples"
+weight: 1
+---
+
+Here will go some basic examples of using TinyGo in the most common scenarios.
+
+### Building "Hello, World" program for WebAssembly
+
+To build the WASM example, run the following command:
+
+```shell
+tinygo build -o wasm.wasm -target=wasm examples/wasm
+```
+
+OK, so what do you do now? Some details from the community here please.
+
+### Building/flashing a "blink" program for micro:bit
+
+Info here...
+
+### Building/flashing a "blink" program for Arduino
+
+To build and then flash a basic blink program for an Arduino Uno, plug in the board to your computer's USB port and then run the following command:
+
+```shell
+tinygo flash -target=arduino examples/blinky1
+```
+
+The built-in LED on the board should start to blink.
+
+### Building/flashing a "blink" program for Circuit Playground Express
+
+To build and then flash a basic blink program for an Circuit Playground Express, 
+
+- Plug your Circuit Playground Express into your computer's USB port.
+- Press the "RESET" button on the board two times to get the Circuit Playground Express board ready to receive code.
```

Comment:
Perhaps an additional hint: the LED ring should first turn red and then green. Green means it is ready to be flashed.